prov/rxd/src/rxd.h

```diff
@@ -427,6 +427,14 @@ void rxd_init_atom_hdr(void **ptr, enum fi_datatype datatype,
 		       enum fi_op atomic_op);
 size_t rxd_init_msg(void **ptr, const struct iovec *iov, size_t iov_count,
 		    size_t total_len, size_t avail_len);
+static inline size_t rxd_check_init_cq_data(void **ptr, struct rxd_x_entry *tx_entry,
+			      		    size_t max_inline)
```

Comment:
I'm not sure what the compiler will do.  So to be safe, pass in &max_inline, and only update it if needed.  The function can convert to void.  Best case is that this saves writing to max_inline when it isn't changing.  The function would end up being almost an exact copy of the code that it is replacing.

Reply:
@shefty Thanks for review. I passed in `&max_inline`, the function was converted in `void` one. Please, check it out.